src/main/webapp/form.html

```diff
@@ -16,15 +16,14 @@ <h3>a recommendation system for students, by students!</h3>
       </div>
     </div>
     <form name="user-details" action="/user" method="POST">
-      <p>Hello! my name is</p>
-      <input type="text" name="name" class="name">
-      <br><br>
-      <p>I am currently a student in the department of</p>
-      <input type="text" name="department" class="department">
-      <br><br>
+      <label for="Name">Hello! my name is</label><br><br>
+      <textarea type="text" id="name" name="name" cols="40" rows="1" required></textarea><br><br>
+      <label for="Department">I am currently a student in the department of</label><br><br>
+      <textarea type="text" id="department" name="department" cols="40" rows="1" required></textarea><br><br>
```

Comment:
I wouldn't use `textarea` for single line small inputs (name and department).

https://stackoverflow.com/q/21698065

Reply:
thank you for the reviews Kazi! will change it :D

Reply:
Hi Kazi, I tried changing it to input and it made a slight change to the UI which I think is worth to consider. Here is the UI with input element
![image](https://user-images.githubusercontent.com/41379082/91172101-8ad62780-e6d3-11ea-8920-8006451eb8ca.png)
please let me know if I should change it or not